src/ui/MessageContent/index.tsx

```diff
@@ -38,6 +38,10 @@ import { noop } from '../../utils/utils';
 import MessageProfile, { MessageProfileProps } from './MessageProfile';
 import MessageBody, { MessageBodyProps } from './MessageBody';
 import MessageHeader, { MessageHeaderProps } from './MessageHeader';
+import Icon, { IconTypes } from '../Icon';
+import FeedbackIconButton from '../FeedbackButton';
```

Comment:
```suggestion
import FeedbackButton from '../FeedbackButton';
```

Reply:
oh thanks but its FeedbackIconButton. I changed to below instead.
```
import FeedbackIconButton from '../FeedbackIconButton';

```